rpcd/playbooks/roles/rpc_maas/files/plugins/neutron_service_check.py

```diff
@@ -36,6 +36,8 @@ def check(args):
     # gather nova service states
     if args.host:
         agents = neutron.list_agents(host=args.host)['agents']
+    elif args.fqdn:
+        agents = neutron.list_agents(host=args.fqdn)['agents']
```

Comment:
@cjloader the ``fqdn`` argument isn't part of the ``args`` ``NameSpace`` object, and will need to be added using the ``add_argument`` method. Similar to what is done for the ``--host`` flag below. Once that's done, the corresponding MaaS check for this plugin will need to be modified to utilize the argument.